test/unit_tests/aievec_tests/floatxfloat_sel/defines.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+#define IN0_SIZE 1024
```

Comment:
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es31-dont-use-macros-for-constants-or-functions